core/storage/base_model/gae_models.py

```diff
@@ -564,6 +628,7 @@ def has_reference_to_user_id(cls, user_id):
     def create(
             cls, entity_id, version, committer_id, commit_type, commit_message,
             commit_cmds, status, community_owned):
+        # type: (Type[SELF_BASE_COMMIT_LOG_ENTRY_MODEL], Text, int, Text, Text, Text, Union[Dict[Text, Any], List[Dict[Text, Any]], None], Text, bool) -> SELF_BASE_COMMIT_LOG_ENTRY_MODEL
```

Comment:
Any use of Any should have a clear comment or a TODO (probably a TODO in this case). Please fix throughout.

Reply:
Done.